README.rst

```diff
@@ -81,13 +146,95 @@ Creating a Cluster
 Running a Relay
 ---------------
 
+Now, with the Replication Set all set up, you can run a relay process.
+
 ::
 
     pgshovel-relay configuration.yml example
 
+
+The relay takes a yaml configuration file, which is in the following format.
+
+::
+
+    stream:
+        path: module.path.to:WriterObject
+        configuration:
+            key: value
+            key2: value2
+
+
+The Relay will attempt to load the Writer object defined at path, calling ``.configure(configuration)`` with the ``configuration`` being the keys and values defined via the ``configuration`` key.  This ``configure`` method is responsible for returning the newly constructed writer instance.
```

Comment:
I think it's fine to be in the same file (for now at least), I just think putting it in a separate section would be better.